streaming/multimodal/convert/laion/laion400m/convert_and_upload.py

```diff
@@ -181,11 +182,11 @@ def upload(local: str, remote: str) -> None:
         local (str): Path on local filesystem.
         remote (str): Path on remote filesystem.
     """
-    local = local.replace(' ', '\\ ')
-    remote = remote.replace(' ', '\\ ')
-    cmd = f'aws s3 cp {local} {remote}'
-    if os.system(cmd):
-        raise RuntimeError(f'Download failed: {cmd}.')
+    local_dir = os.path.dirname(local)
```

Comment:
Does this require local path to contain a directory? Can we pass "." or something